### PR TITLE
[Cloudflare for SaaS] Combined issue and validate certificate pages

### PR DESCRIPTION
**Pull request by mmalden**

### CODE REVIEW

File: content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md [hunk 1]
```diff
@@ -1,125 +1,149 @@
...
-### HTTP
-
+ 
+#### HTTP
+ 
 HTTP adds a DCV token to your origin. You can either add that token [manually](#http-manual) to support pre-validation or wait for Cloudflare to add the DCV token [automatically](#http-automatic), which may lead to a few minutes of downtime.
-
+ 
 {{<Aside type="warning">}}
+<<<<<<< HEAD:content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md
```
Comment (ranbel) [inline content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md]:
```suggestion
```

File: content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md [hunk 2]
```diff
@@ -1,125 +1,149 @@
...
+ 
+HTTP DCV validation is no longer allowed for wildcard certificates. For more details and next steps, refer to [DCV Validation](/cloudflare-for-saas/ssl/common-tasks/issue-and-validate/#dcv-methods).
+ 
+=======
```
Comment (ranbel) [inline content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md]:
```suggestion
```

File: content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md [hunk 3]
```diff
@@ -1,125 +1,149 @@
...
 
 HTTP DCV validation is not allowed for wildcard certificates. For more details and next steps, refer to [Changes to HTTP DCV](/ssl/ssl-tls/migration-guides/dcv-update/).
```
Comment (ranbel) [inline content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md]:
```suggestion
```

File: content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md [hunk 4]
```diff
@@ -1,125 +1,149 @@
...
 
+>>>>>>> production:content/cloudflare-for-saas/ssl/common-tasks/certificate-validation-methods.md
```
Comment (ranbel) [inline content/cloudflare-for-saas/ssl/common-tasks/issue-and-validate.md]:
```suggestion
```

**Review by ranbel — APPROVED** [review]

**Review by Princeobinna — COMMENTED** [review]
Approve assets

**Pull request merged**